tempesta_fw/server.h

```diff
@@ -79,10 +81,11 @@ typedef struct {
  */
 struct tfw_srv_group_t {
 	struct list_head	list;
+	struct list_head	list_reconfig;
```

Comment:
Please comment the item above. Need to answer why `list` isn't enough, i.e. when a group can be referenced from `sg_list` and be in `sg_list_reconfig` at the same time.